main.go

```diff
@@ -5,6 +5,7 @@ import (
 	"net/http"
 
 	"github.com/brigadecore/brigade-cloudevents-gateway/internal/cloudevents"
+	ourCloudHTTP "github.com/brigadecore/brigade-cloudevents-gateway/internal/cloudevents/http" // nolint: lll
```

Comment:
Yes... `ourCloudHTTP` is an ugly name, but we're working with four different `http` packages here, and this alias gets the point across. 🤷‍♂️